### DISCUSSION TITLE
How to call Angular Injectable directly in Angular Stories

### DISCUSSION THREAD

**Discussion by Deepads** [storybookjs/storybook#28504] [Help]
### Summary

How to call Angular Injectable directly in Angular Stories.
Do we have any way to inject the services or other injectables directly in the angular service?

**Comment by Marklb** [comment]
Can you provide more information about what you are asking?

The rendered story is an Angular application. So, your story can access anything in the injector.

If you are wanting to access the injector for a Storybook decorator or render function then that is not possible, because the injector does not exist, yet. If you are just wanting to add providers then you can add them at the application or module level with the Storybook decorators `applicationConfig({ providers: [] })` or `moduleMetadata({ providers })`.
